### PR TITLE
Bug 1837675: Backport enhancements/oc/mirroring-release-signatures implementation

### PR DESCRIPTION
**Pull request by jottofar**
Backport https://github.com/openshift/oc/pull/343 "Implement enhancements/oc/mirroring-release-signatures". Reference https://issues.redhat.com/browse/OTA-210.

### CODE REVIEW

**Comment by wking** [comment]
You also want #392, right?

**Comment by jottofar** [comment]
> You also want #392, right?

Yes I do :) Thanks.

File: go.mod [hunk 1]
```diff
@@ -33,10 +33,10 @@ require (
 	github.com/moby/buildkit v0.0.0-20181107081847-c3a857e3fca0
 	github.com/mtrmac/gpgme v0.1.2 // indirect
 	github.com/opencontainers/go-digest v1.0.0-rc1
-	github.com/openshift/api v0.0.0-20200205133042-34f0ec8dab87
+	github.com/openshift/api v0.0.0-20200210091934-a0e53e94816b
 	github.com/openshift/build-machinery-go v0.0.0-20200211121458-5e3d6e570160
 	github.com/openshift/client-go v0.0.0-20200116152001-92a2713fa240
-	github.com/openshift/library-go v0.0.0-20200206134157-b4c763d94dcf
+	github.com/openshift/library-go v0.0.0-20200303185131-81598fff9efa
```
Comment (soltysh) [inline go.mod:39]: This does not exist in https://github.com/openshift/library-go/tree/release-4.4/.
You need to backport the necessary library-go changes to 4.4 branch too. 
/hold

**Comment by soltysh** [comment]
This is waiting for https://github.com/openshift/library-go/pull/800

File: go.mod (hunk 1 above)
Comment (jottofar) [reply go.mod]: fixed

**Comment by ashcrow** [comment]
Reading through this sounds like it's a feature backport that includes:

- https://github.com/openshift/oc/pull/343
- https://github.com/openshift/oc/pull/392
- Vendor lib bumps

is this accurate?

**Comment by jottofar** [comment]
> Reading through this sounds like it's a feature backport that includes:
> 
>     * #343
> 
>     * #392
> 
>     * Vendor lib bumps
> 
> 
> is this accurate?

Yes. The vendor bump was minimal as driven by the change to library-go.

**Comment by sdodson** [comment]
/lgtm cancel
Needs rebase which will require a new lgtm anyway.

**Comment by ashcrow** [comment]
Removing cherry-pick-approved until rebase and retest.

**Comment by sdodson** [comment]
I've reapplied the label now that it's been rebased.

**Pull request merged**